### PR TITLE
chore(agent): update agent template utilities

### PR DESCRIPTION
**Pull request by tthvo**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] Signed the last commit: `git commit --amend --signoff`
_______________________________________________

Depends on https://github.com/cryostatio/cryostat-core/pull/241
Related to https://github.com/cryostatio/cryostat-core/issues/239

## Description of the change:

- Update to use the latest core utilities.
- Remove unused field in notifications (i.e. previously it holds the temporary uploaded filepath).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@
   <com.google.dagger.version>2.45</com.google.dagger.version>
   <com.google.dagger.compiler.version>2.45</com.google.dagger.compiler.version>
 
-  <io.cryostat.core.version>2.21.0</io.cryostat.core.version>
+  <io.cryostat.core.version>2.22.0</io.cryostat.core.version>
```
Comment (maxcao13) [inline pom.xml:86]: Requires a -core bump?

Comment (tthvo) [reply pom.xml:86]: yep updated from `2.21.0` to `2.22.0`. Just waiting for -core to release `2.22.0` version :))

Comment (andrewazores) [reply pom.xml:86]: `2.21.1` should be available now.

Comment (tthvo) [reply pom.xml]: Opps thanks! Updated now!

Comment (andrewazores) [reply pom.xml]: No worries, I just did that earlier this morning.

**Review by andrewazores — APPROVED** [review]

**Pull request merged**